### PR TITLE
JAV-132 unit test optimize and resolve compile warnings

### PR DESCRIPTION
**Pull request by wujimin**
unit test optimize for fast "mvn install"
now cost 3:51, and can be reduced to 1:21 in the further

### CODE REVIEW

File: demo/demo-spring-boot-discovery/demo-spring-boot-discovery-client/src/test/java/io/servicecomb/demo/discovery/client/DiscoveryClientIT.java [hunk 1]
```diff
@@ -13,43 +13,44 @@
...
 
+import io.servicecomb.springboot.starter.provider.EnableServiceComb;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = RANDOM_PORT, classes = DiscoveryClientIT.DiscoveryTestApplication.class)
 public class DiscoveryClientIT {
 
-	@Autowired
-	private DiscoveryClient discoveryClient;
+    @Autowired
```
Comment (seanyinx) [inline demo/demo-spring-boot-discovery/demo-spring-boot-discovery-client/src/test/java/io/servicecomb/demo/discovery/client/DiscoveryClientIT.java]: could you use the latest code style in /etc? we standardized to google style

**Comment by WillemJiang** [comment]
I just have quick question. How did reduce the "mvn clean install " time,  I didn't find the key change here.

**Comment by wujimin** [comment]
this  pr  is just part of  prepare,  after all code committed, time will  be  reduced

**Comment by wujimin** [comment]
at last reuseFork will  be changed from false to  true in pom , and  then  time reduced

**Pull request merged**